openlibrary/plugins/openlibrary/js/Toast.js

```diff
@@ -0,0 +1,30 @@
+import $ from 'jquery';
+
+const DEFAULT_TIMEOUT = 5000;
+
+
+export class Toast {
```

Comment:
Is Toast.js a testing class? Is this related to our modals?

Reply:
> A toast component was created to provide feedback when a note is updated. This component is stickied near the top of the screen, and is removed after 5 seconds (it can also be closed manually by patrons). When observations are updated and the modal is closed, the throbber is displayed in the observation view and updated observations are fetched from the database.

never mind, I see! TIL: toast messages :)

Reply:
That's what they are called in Android.  I'm not sure if this type of component is known as a toast message in a web development context, though...

I've noticed that these toast components are displacing other elements on the page, so I'll need to figure out how to keep that from happening.  I also think that a target element in which to mount the toast message to should be passed in the constructor.  This way, we can be sure that the toast message is visible when the modal is open (by mounting it to the modal).

If I can't figure out how to display the messages without disrupting the page layout, I may remove this entirely and provide some other type of feedback for note submissions (maybe replacing the submit button's text with a spinner while waiting for a response from the server).